### PR TITLE
ODATA-1498

### PR DESCRIPTION
**Pull request by HeikoTheissen**

### CODE REVIEW

**Comment by ralfhandl** [comment]
The intermediate file https://www.oasis-open.org/apps/org/workgroup/odata/download.php/70018/odata-json-format-v4.01-errata01.docx contained an additional paragraph in section 4.5.3 Control Information: type

> The type control information can be absent in properties nested in an instance of type `Edm.Untyped`. In particular, individual primitive values within a collection cannot have type control information.

Is this no longer necessary?

**Comment by HeikoTheissen** [comment]
I have added it back, and also added the proposal made "after reopening".

File: odata-json-format/7 Structural Property.md [hunk 1]
```diff
@@ -145,6 +145,10 @@ Example ##ex: partial collection of strings with next link
 ```
 :::
 
+A collection of primitive values that occurs in a property of type `Edm.Untyped`
+is interpreted as a collection of `Edm.Boolean`, `Edm.String` and `Edm.Decimal`,
+depending on the Javascript type.
```
Comment (ralfhandl) [inline odata-json-format/7 Structural Property.md:150]:
```suggestion
depending on the JavaScript type.
```
Use same spelling as in all four other places

**Review by ralfhandl — CHANGES_REQUESTED** [review]
Minor nit

**Review by ralfhandl — DISMISSED** [review]

**Review by mikepizzo — APPROVED** [review]

**Pull request merged**